LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2019, 2020 Adfinis SyGroup AG (adfinis-sygroup.ch)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Can't just call COPYING.LESSER -> LICENSE? That is strange.

Reply:
I have all three files.

* **COPYING** contains the GPL license one has to ship.
* **COPYING.LESSER** contains the LGPL amendments one has to ship.
* **LICENSE** details that the software is licensed under LGPL and who holds the copyright.

But I just realised that the text from the LICENSE file is not correct. I have to change that for the **L**GPL one.

Reply:
Sorry. That didn't really answer your question. At least not directly.

From what I gather, having the copyright notice in the read-me would be enough. I'm not sure about the naming. COPYING.LESSER does seem to be a suggestion/recommendation.

I would vote against COPYING and LICENSE because this takes them apart in the directory listing. It is also not as immediately apparent that/how the content of those files is related.

Plus, the [GitHub guide for licensing repositories](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#determining-the-location-of-your-license) has a [recommended example](https://github.com/hubotio/hubot/blob/master/LICENSE.md) that sports a license disclaimer in a LICENSE.md.

Reply:
I'll kill the LICENSE file and rename the COPYING.LESSER.

Reply:
Sorry that these guides all differ, there is always the Nth+1 standard, therefor we continue our (and the common) practice. 

We need only one file. LICENSE with the GPL License text in it. Nothing else: no mention in the README, not extra COPY* files, and definitely no LICENSE.md.

Here some examples:

https://github.com/adfinis-sygroup/sbb-fallblatt
https://github.com/adfinis-sygroup/vault-client
https://github.com/adfinis-sygroup/timed-backend
https://github.com/adfinis-sygroup/mariadb-ansible-galera-cluster

If you really want to do more then you can use [reuse](https://fsfe.org/activities/reuse/reuse.en.html) like in caluma. But some of the js-folks already made it clear, that they don't like the python-dependency. That is valid argument, so I suggest:

* Only the LICENSE file containing the original plain text license

* You don't have to remove mention in the README if it is only one sentence like: The project is licensed under GPL-3.0-or-later

* Any mention of the license should use the [SPDX](https://spdx.org/licenses/) short identifier

* Unfortunately github does not recognize the SPDX version of the license so you have to use the official plain text license: https://www.gnu.org/licenses/gpl-3.0.txt

Reply:
Hmmm... Shouldn't this be LGPL'd?

But I could create a LICENSE file that includes the GPL and the LGPL additions on after another.

Reply:
No. If it LGPL, only add the complete LGPL license to LICENSE:

https://www.gnu.org/licenses/lgpl-3.0.txt

The sort indentifier is: LGPL-3.0-or-later

Reply:
According to the [guide](https://www.gnu.org/licenses/gpl-howto.html) both need to be included:

> If you are releasing your program under the Lesser GPL, you should also include the text version of the LGPL, usually in a file called COPYING.LESSER. **Please note that, since the LGPL is a set of additional permissions on top of the GPL, it's crucial to include both licenses** so users have all the materials they need to understand their rights.

Reply:
@fkm ok, got it, sorry for not getting that earlier, this is super stupid. I refuse to do this, so I have to think about it. You'll hear to me in 2-3h.

Reply:
Yeah. Having a split license seemed weird to me too :smiley: 

Take your time. There is no release planned. As long as we find something that is correct (enough) and can be used everywhere. This PR went overboard long ago anyway :smile: